source/js/components/donation-amount-toggle.js

```diff
@@ -0,0 +1,30 @@
+class AmountToggle {
+  static selector() {
+    return "[data-amount-toggle]";
+  }
+
+  constructor() {
+    this.toggleButton = document.querySelector("[data-amount-toggle]");
```

Comment:
rather than re-selecting `[data-amount-toggle]` you can just pass `node` into the constructor and assign it to `this.toggleButton`, e.g.

```
constructor(node) {
    this.toggleButton = node;
```